Documentation/doc/scripts/compare_testsuites.sh

```diff
@@ -40,8 +40,8 @@ cd ..
 echo "Output generated"
 if ! [ -d "$DOC_REF" ]; then
   echo "No reference given. Script is finished."
-  exit 2
+  exit 0
```

Comment:
@maxGimeno It seems that exit is a normal exit of the script. That is why I have change the exitcode to 0.